db/migrate/20231118044848_add_consultations_received_to_patients.rb

```diff
@@ -0,0 +1,5 @@
+class AddConsultationsReceivedToPatients < ActiveRecord::Migration[7.0]
+  def change
+    add_column :patients, :consultations_received, :integer
```

Comment:
This value will be defined by the number of consultations entries; which is something we have pending to do
The use needs to be able to add as many entries as needed; e.i patient has many consultations

Roll this back for now

Reply:
done!